File: Pratica 6 - Funcoes 2/java_logic/Exercicio2_v2.java
import java.util.Scanner;

public class Exercicio2_v2 {
    static double calcular_area(double base, double altura) {
        return base * altura;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Digite a base do retângulo:");
        double base = sc.nextDouble();
        System.out.println("Digite a altura do retângulo:");
        double altura = sc.nextDouble();

        double area = calcular_area(base, altura);
        System.out.println("A área do retângulo é: " + area);

        sc.close();
    }
}

File: Pratica 6 - Funcoes 2/cpp_logic/exercicio2_v2.cpp
#include <iostream>
using namespace std;

float area(float base, float altura) {
    return base * altura;
}

int main() {
    float base, altura;
    cout << "Digite a base: ";
    cin >> base;
    
    cout << "Digite a altura: ";
    cin >> altura;

    cout << "Área: " << area(base, altura) << endl;

    return 0;
}